css/style.css

```diff
@@ -1,3 +1,1033 @@
+  }
+
+  .product-mandel_g:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-mandel_b {
+    position: absolute;
+    display: inline-block;
+    right: 60px;
+    bottom: 30px;
+    background: #762e31;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-mandel_b:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-mandel_bl {
+    position: absolute;
+    display: inline-block;
+    right: 45px;
+    bottom: 30px;
+    background: #41479b;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-mandel_bl:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-mandel_y {
+    position: absolute;
+    display: inline-block;
+    right: 30px;
+    bottom: 30px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-mandel_y:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  /* cloud */
+
+  .products__product-cloud {
+    display: block;
+    align-self: flex-start;
+    text-align: left;
+    position: relative;
+    width: 208px;
+    height: 261px;
+    left: 0;
+    top: 15px;
+  }
+
+  .product-cloud_g {
+    position: absolute;
+    display: inline-block;
+    right: 75px;
+    bottom: 30px;
+    background: #c4c4c4;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_g:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_b {
+    position: absolute;
+    display: inline-block;
+    right: 60px;
+    bottom: 30px;
+    background: #257da9;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_b:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_r {
+    position: absolute;
+    display: inline-block;
+    right: 45px;
+    bottom: 30px;
+    background: #b62e3c;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_r:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .product-cloud_y {
+    position: absolute;
+    display: inline-block;
+    right: 30px;
+    bottom: 30px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 7px;
+    height: 7px;
+    transition: transform 1s linear;
+  }
+
+  .product-cloud_y:hover {
+    transform: scale(1.2, 1.2);
+  }
+
+  .products__img {
+    display: block;
+    width: 169.89px;
+    height: 173.07px;
+    margin: 0 auto 23.67px auto;
+  }
+
+  .products__name {
+    font-size: 12px;
+    top: 185px;
+  }
+
+  .products__cost {
+    font-size: 14px;
+    top: 215px;
+  }
+}
+
+/* desktop 1170 */
+
+@media all and (min-width: 1170px) {
+  .page {
+    max-width: 1170px;
+  }
+
+  .page__header {
+    max-width: 1170px;
+    margin: 0 auto 20.17px auto;
+ }
+
+  .menu__left {
+   width: 40%;
+   left: 52px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)